tox.ini

```diff
@@ -25,7 +25,7 @@ deps =
      -r{toxinidir}/requirements_test.txt
      -c{toxinidir}/homeassistant/package_constraints.txt
 commands =
-     pylint homeassistant
+     pylint --jobs=4 homeassistant
```

Comment:
Another option would be to set jobs to 0 which will use the total number of CPUs according to the docs.